README.md

```diff
@@ -1,24 +1,234 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
+  <img src="./draft.png" alt="logo" width="256"  height="auto" />
```

Comment:
- [ ] I would advise you to remove the broken logo or fix it if you can, this is because you need a clean Readme template. It shows professionalism and confidence.